### PR TITLE
Added Weapons, Edited Verbs, et cetera

### PR DESCRIPTION
**Pull request by ValerieSteel**
## About The Pull Request
Added two new W-Corp gacha weapons: the Modified W-Corp Hatchet, which can spend charge to cripple the targeted enemy, and the Modified W-Corp Warhammer, which spends charge on lowering their BLACK resist. Also, I edited some verbs to make some EGO weapons a tad more flavorful.

## Why It's Good For The Game
More W-Corp stuff is good! Also, the verbs I chose are neat, I think. They'd really add some minor flavor to the game.

## Changelog
:cl:
add: Added 2 new weapons to the W-Corp refinery gacha. Icons/sprites are currently placeholders.
tweak: Edited several verbs for some EGO weapons to make things more flavorful.
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/aleph.dm [hunk 1]
```diff
@@ -345,8 +345,8 @@
 	force = 70	//there's a focus on the ranged attack here.
 	damtype = BLACK_DAMAGE
 	armortype = BLACK_DAMAGE
-	attack_verb_continuous = list("attacks")
-	attack_verb_simple = list("attack")
+	attack_verb_continuous = list("CENSOREDs")
```
Comment (Coxswain-Navigator) [inline code/game/objects/items/ego_weapons/aleph.dm:348]: Really? No.

File: code/game/objects/items/ego_weapons/waw.dm [hunk 2]
```diff
@@ -870,8 +870,8 @@
 	attack_speed = 1.2
 	damtype = RED_DAMAGE
 	armortype = RED_DAMAGE
-	attack_verb_continuous = list("pokes", "jabs", "tears", "lacerates", "gores")
-	attack_verb_simple = list("poke", "jab", "tear", "lacerate", "gore")
+	attack_verb_continuous = list("pokes", "jabs", "tears", "lacerates", "gores", "gazes upon")
```
Comment (Coxswain-Navigator) [inline code/game/objects/items/ego_weapons/waw.dm:873]: This is bad, just take it out.

Comment (ValerieSteel) [reply code/game/objects/items/ego_weapons/waw.dm:873]: No need to be a dick about it, man. Yeesh.

**Comment by Kitsunemitsu** [comment]
Weapons were added in #1032

**Pull request closed without merging**